docs/data/sdks/next-gen-sdks.md

```diff
@@ -0,0 +1,186 @@
+
+## Common methods
+
+Next-gen SDKs share the same architecture as well as the same interfaces. Aligned interfaces benefit using Amplitude SDKs on multiple platforms without memorizing different function names.
+
+This a list of shared interfaces of next-gen SDKs. 
+
+!!!note 
+    Different SDKs follow different naming conventions. Here use camel case for methods and parameters and use pascal case for classes. 
+
+- `track(BaseEvent)`
+- `identify(IdentifyEvent)`
+- `setGroup(groupType, groupName)`
+- `groupIdentify(groupType, groupName, IdentifyEvent)`
+- `revenue(RevenueEvent)`
+- `flush()`
+- `add(Plugin)`
+- `remove(Plugin)`
+- `shutdown()`
+
+## Next-gen benefits 
+
+- Next-gen SDKs uses **modern languages** for each platform and with **modern tools/frameworks**
+    - It's easier for develpers to onboard Amplitude and also gain more confidence on data quality
+    - Next-gen Browser, Node.js, React Native SDK use TypeScript. Next-gen Android uses Kotlin. Next-gen iOS uses Swift
+    - Improved the code readability can speed up your the developement cycle
+    - Browser SDK implements tree-shaking technique to reduce the bundle size
+
+- Next-gen SDKs are designed with a solid and **extensible architecture**
+    - Next generation SDKs allow more customization based on your needs. You can implement a custom logger, enrich your data, and send your data to other destination by using Plugins
+
+- Next-gen SDKs has **cleaner and aligned interfaces**
+    - Aligned interfaces benefit using Amplitude SDKs on multiple platforms without memorizing different function names. For example, before next-gen SDKs, to track an event with different properties, you need to use different functions, for example `logEvent()`, `logEventWithTimestamp()`, `logEventWithGroup()`, etc. Right now with next-gen SDKs, you can just use by `track()` with all properties in `EventOptions`
+    
+- Next-gen SDKs are **more reliable and robust**
+    - Next-gen SDKs don't require any external dependencies, which enhances performance, mitigate potential issues caused by dependencies, and makes them more lightweight
+    - Next-gen SDKs use file storage instead of database to improve performance
+    - Next-gen Android SDK removes OkHttp which addresses dependency compatibility issues
+
+## Comparison with maintenance SDK
+
+If you are migrating from maintenance SDKs, you may notice that next-gen SDKs differ from maintenance SDKs not only in terms of interfaces but also in their Middleware vs Plugin architecture . You can think of Middleware as being equal to Enrichment Plugin and Destination Plugin.
+
+## SDK status table
+
+|Platform|Next-gen|Maintenance|Which SDK (repo name)|SDK Package|
+|--------|---|---|---------------------|-----------|
+|[Browser](./javascript/index.md)|:material-close:|:material-check:|[Amplitude-JavaScript](https://github.com/amplitude/Amplitude-JavaScript)|`@amplitude/amplitude-js`|
```

Comment:
use the `{ .green }` or `{ .red }` suffix on the `:material-check:` and `:material-close:` icons, respectively to modify the color to make this table more visual.

```suggestion
|[Browser](./javascript/index.md)|:material-close:{ .red }|:material-check:{ .green }|[Amplitude-JavaScript](https://github.com/amplitude/Amplitude-JavaScript)|`@amplitude/amplitude-js`|
```

Reply:
That's awesome. Update the table and make maintenance SDK icon red.